rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -1233,85 +1230,6 @@ def test_optimism_giveth_staked_balances(
     )
 
 
-@pytest.mark.vcr(filter_query_parameters=['apikey'])
-@pytest.mark.parametrize('should_mock_current_price_queries', [False])
-@pytest.mark.parametrize('ethereum_manager_connect_at_start', [(INFURA_ETH_NODE,)])
-@pytest.mark.parametrize('ethereum_accounts', [['0x561fe975d925CC259D5aFF7A4d83612Fb4758103']])
```

Comment:
the v3 balances are queried now as simple tokens? can you run a test where you detect them? probably you can patch the get_tokens function from the globaldb to return only the uniswap v3 token. 

One problem: when do we detect the v3 token? Have you tested it in real life with the app? The whole flow I mean. In theory it should work but please test it with the app

Reply:
> yep, used a completely fresh data dir. Looks like it works fine. Decoded the events and then redetect tokens/refresh balances and the positions are shown correctly.